apps/api-extractor-model/src/model/ModelReferenceResolver.ts

```diff
@@ -102,7 +102,7 @@ export class ModelReferenceResolver {
         return result;
       }
 
-      const foundMembers: ReadonlyArray<ApiItem> = currentItem.findMembersByName(identifier);
+      const foundMembers: ReadonlyArray<ApiItem> | undefined = currentItem.findMembersByName(identifier);
```

Comment:
Why `| undefined` here?

Reply:
left over from refactoring `ApiTypeParameter` to just `TypeParameter`.